src/main/java/edu/ucsb/cs56/ucsb_courses_search/controller/CSVDownloadController.java

```diff
@@ -20,13 +20,15 @@
     @Autowired
     private CurriculumService curriculumService;
 
+    String filename = "courses.csv";
```

Comment:
Because this is a constant that only matters to this class, this should be a `private static` variable. Constants are also generally given a name in all caps (e.g. `FILENAME`).

Reply:
I would recommend not using a static name for the file at all. It would be better to descriptively name the csv based on the requested parameters, such as quarter, subject area, etc.

Reply:
I agree that the csv naming convention could be more detailed. However, I think we can tackle the issue once we start to add the csv downloads for each page.

Reply:
Hm, weird error when I try to specify the csv file name:
`String filename = subjectArea + "-" + quarter + "-" + courseLevel;`

Doing this will actually reproduce the error that this branch originally fixed.

Reply:
@troylee24 
You might try: 
`String filename = subjectArea + "-" + quarter + "-" + courseLevel + ".csv";`